apps/src/makerlab/TouchSensor.js

```diff
@@ -0,0 +1,64 @@
+/** @file Wrapper for playground-io Touchpad to expose individual pads */
+import {EventEmitter} from 'events'; // provided by webpack's node-libs-browser
+
+/**
+ * Given a playground-io Touchpad controller, monitors touch events and forwards
+ * 'down' and 'up' events for a particular pin.
+ * @constructor
+ * @param {number} pinIndex - Which pin to pay attention to and forward events for.
+ * @param {Touchpad} touchpadsController - playground-io Touchpad controller.
+ */
+export default class TouchSensor extends EventEmitter {
+  constructor(pinIndex, touchpadsController) {
+    super();
+
+    /**
+     * @private {number} Which single pin this controller will monitor and
+     *          forward events for.
```

Comment:
I know I don't indent like that (i just indent the next line two spaces). I also don't think it's that big of a deal if our styles are slightly different in this respect.